test/testutil/testutil.go

```diff
@@ -335,3 +335,12 @@ func GetCreationTimestamp() string {
 func GenerateS3BucketName() string {
 	return "radiusfunctionaltestbucket-" + uuid.New().String()
 }
+
+func GetDockerRegistryName() string {
```

Comment:
How is this different from `DOCKER_REGISTRY`? Looks like they are two different settings with the same default value. Maybe a comment would help?

Reply:
I added a comment. In the case of using a local registry, DOCKER_REGISTRY will return the port too, so I added this function that returns the name of the registry without a port if present.